docs/RationaleHW-SV.md

```diff
@@ -388,6 +388,28 @@ still many things in SystemVerilog that we cannot currently express in the SV
 dialect, this design makes it easy to incrementally build out new capabilities
 over time.
 
+#### Verbatim op
+The verbatim operation produces a typed value expressed by a string of
+SystemVerilog.  This can be used to access macros and other values that are
+only sensible as Verilog text.
+
+The text string is expected to have the highest precedence, so you should
+include parentheses in the string if it isn't a single token.  This is also
+assumed to not have side effects (use sv.verbatim.expr.se) if you need them.
```

Comment:
I'd recommend backquoting some of the things here like `sv.verbatim.expr.se` and `{{0}}` in the markdown